spinalcordtoolbox/reports/qc2.py

```diff
@@ -48,13 +48,16 @@
 # (the height is automatically adjusted based on the aspect ratio of the image)
 # Notes:
 #   - This shouldn't ever need to be changed, since "target width" is related
-#     to the design of the QC report interface. If things need to be scaled,
-#     it should be done to the array itself before it is written to the .
+#     to the design of the QC report interface. The images are currently rendered
+#     at 1060px wide, so we should match that in the mosaic arrays we generate.
+#     NOTE: This will result in arrays smaller than 1060px wide, since mosaic
+#     gird wrapping won't exceed this value. But, a smaller visible mosaic is
+#     worth the trade-off of pixel accuracy when rendering the final image.
 #   - `matplotlib` uses inches/DPI to define the canvas. But, given we want
 #     a fixed output size, we can choose some arbitrary values for both.
 #     Presumably, choosing a different DPI value (and thus a different canvas
 #     size in inches) wouldn't change the output image at all. (Is this true?)
-TARGET_WIDTH_PIXL = 1500
+TARGET_WIDTH_PIXL = 1060
```

Comment:
Additionally, skimming through all of the reports, we do see the result of changing `TARGET_WIDTH_PIXL` -- now the axial mosaics are rendered at full size (instead of being scaled down a little):

<img width="1920" height="915" alt="image" src="https://github.com/user-attachments/assets/c6a7c789-57eb-4f03-bc36-559d5c692fc4" />

After:

<img width="1920" height="915" alt="image" src="https://github.com/user-attachments/assets/b9528753-a31f-4fb4-bf67-144d164cfb1c" />

Personally I find this easier on the eyes, but I could understand if we wanted to keep the scaled-down version.